frontend/src/metabase/common/components/CollectionPicker/components/PersonalCollectionItemList.tsx

```diff
@@ -48,6 +48,7 @@ const getSortedTopLevelPersonalCollections = (
     .map(
       (collection: Collection): CollectionPickerItem => ({
         ...collection,
+        here: ["collection"], // until this endpoint gives this to us, pretend they all have content
```

Comment:
This is the one place where you can end up being tricked into looking into a collection without children. This endpoint doesn't support there `here` and `below` flags yet, and I kind suspect that it might not be worthwhile - since picking things in other users' personal collections is such an infrequently traveled path